docker-bake.hcl

```
@@ -145,15 +181,19 @@ function "alpine_platforms" {
   params = [jdk]
   result = (equal(21, jdk)
     ? ["linux/amd64", "linux/arm64"]
-  : ["linux/amd64"])
+    : (equal(25, jdk)
+      ? ["linux/amd64", "linux/arm64"]
+      : ["linux/amd64"]))
 }
 
 # Return an array of Debian platforms to use depending on the jdk passed as parameter
 function "debian_platforms" {
   params = [jdk]
   result = (equal(17, jdk)
     ? ["linux/amd64", "linux/arm64", "linux/ppc64le", "linux/arm/v7"]
-  : ["linux/amd64", "linux/arm64", "linux/ppc64le", "linux/s390x"])
+    : (equal(21, jdk)
+      ? ["linux/amd64", "linux/arm64", "linux/ppc64le", "linux/s390x"]
+      : ["linux/amd64", "linux/arm64", "linux/ppc64le", "linux/s390x"]))
```

```suggestion
    : ["linux/amd64", "linux/arm64", "linux/ppc64le", "linux/s390x"]))
```
todo: only JDK17 provides a different set of architectures, no need to check if it's JDK21 then return the same set whether it's JDK21 or JDK25.